kubernetes/services/api-gateway.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: apimanager
+  name: apimanager
+spec:
+  selector:
+    matchLabels:
+      app: apimanager
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: apimanager
+    spec:
+      containers:
+        - image: zhonghaolyu/repo:apiKube4
```

Comment:
would it be better if we store/pull it from our shared account? I can send you the login detail if you want.

Reply:
This is a good idea. It should be a public one if possible.

Reply:
@kevin-zhonghao Can you get the information from @er1cthe0ne and make the changes? Thanks.